includes/class-plugin.php

```diff
@@ -282,7 +282,7 @@ public function enqueue_admin_styles() {
         if ( ! in_array( $screen->id, $screens, true ) ) {
             return;
         }
-
+        /** @phpstan-ignore-next-line */
         wp_enqueue_style( 'redis-cache', WP_REDIS_PLUGIN_DIR . '/assets/css/admin.css', null, WP_REDIS_VERSION );
```

Comment:
@zaherg Could an empty array in place of `null` fix this?
https://developer.wordpress.org/reference/functions/wp_enqueue_style/

Reply:
just pushed the fix, I was reading the error from `wp_enqueue_script` while updating it 🫣 , sorry about that, WP is not my strongest feature